#! /bin/sh
patch -p1 -f $* < $0
exit $?

commit eb08800c23ec64330e3ad415141ecb6e686039e4
Author: Andrej Kacian <ticho@claws-mail.org>
Date:   Tue Jan 19 15:01:50 2016 +0100

    Fix typo in handling feed name ending with a period on Win32.

diff --git a/src/plugins/rssyl/rssyl_subscribe.c b/src/plugins/rssyl/rssyl_subscribe.c
index b9dbc43..f80ec82 100644
--- a/src/plugins/rssyl/rssyl_subscribe.c
+++ b/src/plugins/rssyl/rssyl_subscribe.c
@@ -133,7 +133,7 @@ gboolean rssyl_subscribe(FolderItem *parent, const gchar *url,
 	if (tmpname2[0] == '.')
 		tmpname2[0] = "_";
 	if (tmpname2[strlen(tmpname2) - 1] == '.')
-		tmpname2[strlen(tmpname2) - 1] == '_';
+		tmpname2[strlen(tmpname2) - 1] = '_';
 #endif
 
 	while (folder_find_child_item_by_name(parent, tmpname2) != 0 && i < 20) {
